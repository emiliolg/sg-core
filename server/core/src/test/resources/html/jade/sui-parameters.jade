doctype html
html(sg-view="tekgenesis.service.html.Views", sg-params="text:String, number:Int, decimal:Decimal, bool:Boolean, list:String*")
    head
        title='Parameters examples'
    body
        div='Text: ' + text
        div='Number: ' + number
        div='Decimal: ' + decimal
        div='Currency: ' + sg.currency(decimal)
        div='Bool: ' + bool
        div='String list: ' + list