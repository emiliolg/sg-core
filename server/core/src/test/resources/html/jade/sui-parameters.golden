<!DOCTYPE html>
<html sg-view="tekgenesis.service.html.Views" sg-params="text:String, number:Int, decimal:Decimal, bool:Boolean, list:String*">
  <head>
    <title>Parameters examples</title>
  </head>
  <body>
    <div>Text: text</div>
    <div>Number: 2</div>
    <div>Decimal: 1.1</div>
    <div>Currency: $1.10</div>
    <div>Bool: true</div>
    <div>String list: (one, two, three)</div>
  </body>
</html>